### PR TITLE
Pn/feature edit / delete topic comment

### PR DESCRIPTION
**Pull request by nemoipaha**

### CODE REVIEW

**Comment by nemoipaha** [comment]
Please see my pr first, I have conflicts with https://github.com/BinaryStudioAcademy/bsa-2016-forum/pull/186

File: resources/assets/sass/index.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/*
+Error: File to import not found or unreadable: bootstrap.
```
Comment (binaryvo) [inline resources/assets/sass/index.css:2]: ?

Comment (TatsuUkraine) [inline resources/assets/sass/index.css]: This file should be in gitignore

Comment (TatsuUkraine) [inline resources/assets/sass/index.css]: Provably

**Comment by binaryvo** [comment]
Remove empty methods in  models, views and so on

File: config/cloudinary.php [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <?php
 
 return [
-    "cloud_name" => "dmxebkh7h",
-    "api_key" => "726481723843648",
-    "api_secret" => "PyBAFmxpaB3eJsgoOwOghF7ih9Q"
+    // pn api keys
+    "cloud_name" => "duep6j77j",
```
Comment (binaryvo) [inline config/cloudinary.php:5]: Is that right cloudinary config or you replaced it with your credentials and made commit?

**Comment by binaryvo** [comment]
Please check twice your push, resolve all remarks and make PR again

**Pull request closed without merging**